src/lib/u2.js

```diff
@@ -1437,3 +1437,52 @@ foam.CLASS({
     }
   ]
 });
+
+foam.CLASS({
+  package: 'foam.u2',
+  name: 'CSS',
+
+  properties: [
+    {
+      class: 'String',
+      name: 'code'
+    },
+    {
+      name: 'name',
+      factory: function() { return 'CSS-' + this.$UID; }
+    }
+  ],
+
+  methods: [
+    function installInClass(cls) {
+      // There might be several CSS axioms on each class.
```

Comment:
Why would you need to install more than one?  I guess if you had a trait/mixin, then you could have more than one.